tools/getevents/getevents.go

```diff
@@ -0,0 +1,46 @@
+package main
```

Comment:
Why do we want to have a separate binary to get events?

Reply:
it was a easy way to test the new code, but I can strip it off if we don't want it

Reply:
If you have usage for it I do not have a problem, it is just hard for me to imagine situation when I will call this binary instead of `oc get events` :)